nuxeo-core/nuxeo-core-storage-dbs/src/main/java/org/nuxeo/ecm/core/storage/dbs/DBSSession.java

```diff
@@ -364,18 +364,40 @@ protected Document getChild(String parentId, String name) {
     }
 
     protected List<String> getChildrenIds(String parentId) {
+        // We want all children, the filter flag is null
+        Boolean childrenFilter = null;
+        return internalGetChildrenIds(parentId, childrenFilter);
+    }
+
+    protected List<String> getChildrenIds(String parentId, boolean excludeSpecialChildren) {
+        return internalGetChildrenIds(parentId, excludeSpecialChildren);
+    }
+
+    private List<String> internalGetChildrenIds(String parentId, Boolean excludeSpecialChildren) {
```

Comment:
Ok let's clarify this and just have only one new `getChildrenIds` method with two booleans: `boolean excludeRegularChildren, boolean excludeSpecialChildren `. And rename the old one `getAllChildrenIds` (which calls the above one with `false, false`). Then change callers to use the new one.
Also, `protected` please.